### PR TITLE
feat: Add 'envFile' in test configuration

### LINKED ISSUE #1214: Allow `envFile` alternative to `env` in `java.test.config`
The VS Code `launch.json` file allows specifying `envFile` in addition to `env`. However, there does not appear to be a similar mechanism when running tests via this extension. It would be super handy to be able to have 1 file specify the `env` rather than duplicate it in both `.env` and `.vscode/settings.json`.

### PR DESCRIPTION
**Pull request by jdneo**
resolve #1214

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -40,6 +40,9 @@ jobs:
     - name: Build OSGi bundle
       run: npm run build-plugin
 
+    - name: prepublish
+      run: npm run vscode:prepublish
```
Comment (CsCherrYY) [inline .github/workflows/build.yml:44]: Suggest moving this part to script `npm run test` in package.json, to make it more convenient in development.

Comment (jdneo) [reply .github/workflows/build.yml:44]: Make sense

Comment (CsCherrYY) [reply .github/workflows/build.yml:44]: get it

**Review by CsCherrYY — DISMISSED** [review]
please do not forget to update the wiki

**Review by CsCherrYY — APPROVED** [review]

**Pull request merged**